app/models/user.rb

```diff
@@ -17,4 +18,35 @@ class User < ApplicationRecord
             length: {minimum: Settings.digit.length_6}
 
   has_secure_password
+
+  class << self
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost:
```

Comment:
```suggestion
      BCrypt::Password.create string, cost: cost
```
Như thế này chứ em nhỉ

Reply:
Dạ vâng